editor/actions.js

```diff
@@ -444,6 +444,20 @@ export function handleMetaBoxReload( location ) {
 }
 
 /**
+ * Returns an action object used to signify that a meta box finished loading.
+ *
+ * @param {String} location Location of meta box: 'normal', 'side'.
+ *
+ * @return {Object} Action object
+ */
+export function metaBoxLoaded( location ) {
+	return {
+		type: 'METABOX_LOADED',
```

Comment:
If we're treating "meta box" as two distinct words (evidenced by action creator name and other action types in this file), this action type should be `META_BOX_LOADED`